tests/test_python_api.py

```diff
@@ -159,19 +247,20 @@ def test_python_api1():
 
     message = mFed.endpoints["ep2"].get_message()
 
-    assert message.message_id == 55
     assert message.is_valid() is True
-    assert message.data == "random-data"
-    assert message.raw_data == b"random-data"
-    assert len(message.raw_data) == 11
-    assert message.original_destination == ""
-    assert message.original_source == "TestFederate/ep1"
-    assert message.source == "TestFederate/ep1"
-    assert message.time == 1.0
```

Comment:
assert_attributes() seems to also make a decent alternative to having a bunch of assert lines.